packages/docs/docusaurus.config.js

```diff
@@ -70,22 +85,27 @@ const config = {
           },
           {
             type: 'doc',
-            docId: '/api-reference',
+            docId: 'api-reference/index',
             position: 'left',
             label: 'API Reference',
           },
           {
             type: 'doc',
-            docId: '/advanced-customization',
+            docId: 'advanced-customization/index',
             position: 'left',
             label: 'Advanced Customization',
           },
           {
             type: 'doc',
-            docId: '/migration-guides',
+            docId: 'migration-guides/index',
```

Comment:
You may notice that there is no "migration-guides" content in the v3/v4 docs. Docusaurus is smart, so if you click this link while looking at v3, it resolves with fallbacks:
- the v3 version, if it exists,
- an older version, if it exists,
- the latest version

So this would resolve to the v5/"current" migration docs.